Client/Frontend/Popover/PopoverController.swift

```diff
@@ -44,6 +44,9 @@ class PopoverController: UIViewController {
     /// The arrow direction behavior for this popover
     var arrowDirectionBehavior: ArrowDirectionBehavior = .automatic
     
+    /// Whether or not to automatically dismiss the popup when the device orientation changes
+    var dismissesOnOrientationChanged = true
```

Comment:
Yeah I added it as a variable for future proofing rather than just force popover dismissal for every popover.

This may end up never being changed at which point we could probably remove it :)